lib/fork_break/process.rb

```diff
@@ -4,16 +4,19 @@ class << self
       attr_accessor :breakpoint_setter
     end
 
+    attr_reader :return_value
+
     def initialize(debug = false, &block)
       @debug = debug
       @fork = Fork.new(:return, :to_fork, :from_fork) do |child_fork|
         self.class.breakpoint_setter = breakpoints = BreakpointSetter.new(child_fork, debug)
 
         breakpoints << :forkbreak_start
-        block.call(breakpoints)
+        returned_value = block.call(breakpoints)
         breakpoints << :forkbreak_end
```

Comment:
Tested!

Yeah, this is quite handy to have.

```ruby
10] pry(main)> ForkBreak::Process.new{Campaign.last}.finish.wait.return_value
  Campaign Load (1.1ms)  SELECT  "campaigns".* FROM "campaigns"  ORDER BY "campaigns"."id" DESC LIMIT 1
=> #<Campaign:0x007f8ae789ae08
 id: 3,
 name: "Foo",
 description: "Foo",
 starts_at: Thu, 05 Nov 2015 23:00:00 UTC +00:00,
 ends_at: Sun, 06 Dec 2015 08:30:45 UTC +00:00,
 fulfilled_at: nil,
 created_at: Fri, 06 Nov 2015 08:30:45 UTC +00:00,
 updated_at: Fri, 06 Nov 2015 08:30:45 UTC +00:00,
 event_promoter_id: 8,
 venue_id: 7>
```

I cannot `.save` that object indeed, but that's largely unimportant :)